src/components/charts/CandleChart.vue

```diff
@@ -309,7 +309,11 @@ function updateChart(initial = false) {
               if (Array.isArray(value)) {
                 if (value.length > 0 && value[0]) {
                   // Show both value and tag
-                  return `${signal.tooltipPrefix} ${value[0]} ${value[1]?.toString().substring(0, 100) ? `(${value[1]})` : ''}`;
+                  //return `${signal.tooltipPrefix} ${value[0]} ${value[1]?.toString().substring(0, 100) ? `(${value[1]})` : ''}`;
+                  // Show both value and tag, but limit tag length to prevent layout issues
```

Comment:
if we assume the fix to be valid - we'll not keep a comment of the old line around.